osu.Game/Screens/Play/MenuOverlay.cs

```diff
@@ -190,6 +194,46 @@ protected MenuOverlay()
             RelativeSizeAxes = Axes.Both;
         }
 
+        protected override bool OnKeyDown(InputState state, KeyDownEventArgs args)
+        {
+            if (args.Repeat) return false;
+
+            var buttonsAmount = Buttons.Children.Count();
+            if (buttonsAmount == 0) return false;
+
+            foreach (var button in Buttons.Children)
+                button.TriggerOnHoverLost(state);
+
+            switch (args.Key)
+            {
+                case Key.Down:
+                    if (selectedButton == 0 || selectedButton == buttonsAmount)
+                        selectedButton = 1;

```

Comment:
I personally don't like the representation of "no button is selected" as 0. I think it is more intuitive to use -1.